src/scripts.js

```diff
@@ -50,11 +50,11 @@ function createUser() {
 };
 
 function showUserCardData() {
-  userCardID.innerText = `ID: ${user.id}`;
-  userCardName.innerText = `Name: ${user.name}`;
-  userCardAddress.innerText = `Address: ${user.address}`;
-  userCardEmail.innerText = `Email: ${user.email}`;
-  userCardStrideLength.innerText = `Stride Length: ${user.strideLength}`;
-  userCardDailyStepGoal.innerText = `Daily Step Goal: ${user.dailyStepGoal}`;
-  userCardFriends.innerText = `Friends: ${user.friends}`;
+  userCardID.innerText = `${user.id}`;
```

Comment:
`showUserCardData()` was changed so that it no longer included the data labels when manipulating the innerText of the user card.